### [CVE-2015-20107](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2015-20107)
![](https://img.shields.io/static/v1?label=Product&message=n%2Fa&color=blue)
![](https://img.shields.io/static/v1?label=Version&message=n%2Fa&color=blue)
![](https://img.shields.io/static/v1?label=Vulnerability&message=n%2Fa&color=brighgreen)

### Description

In Python (aka CPython) up to 3.10.8, the mailcap module does not add escape characters into commands discovered in the system mailcap file. This may allow attackers to inject shell commands into applications that call mailcap.findmatch with untrusted input (if they lack validation of user-provided filenames or arguments). The fix is also back-ported to 3.7, 3.8, 3.9

### POC

#### Reference
- https://github.com/python/cpython/issues/68966

#### Github
- https://github.com/ARPSyndicate/cvemon
- https://github.com/GitHubForSnap/matrix-commander-gael
- https://github.com/Live-Hack-CVE/CVE-2015-20107
- https://github.com/codeskipper/python-patrol
- https://github.com/flexiondotorg/CNCF-02

